old/claude_processor.py

```
@@ -0,0 +1,197 @@
+                print(f"Warning: Missing required fields: {', '.join(missing_fields)}")
+            
+            return parsed_data
+            
+        except json.JSONDecodeError as e:
+            print("Error: Could not parse JSON from Claude's response")
+            print(f"JSON Error: {str(e)}")
+            print("Raw response:", response_text)
+            print("\nCleaned JSON:", cleaned_json)
+            return None
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)